lib/router.ex

```diff
@@ -1,17 +1,68 @@
 defmodule SituationRoom.Router do
-  import Plug.Conn
   use Plug.Router
+  alias SituationRoom.Site
 
-  plug Plug.Logger
-  plug :match
-  plug :dispatch
+  if Mix.env() == :dev do
+    use Plug.Debugger
+  end
+
+  use Plug.ErrorHandler
+
+  plug(Plug.Logger)
+  plug(:match)
+  # If we are going to include plug.Parsers, they go right here
+  plug(:dispatch)
 
   get "/ping" do
     send_resp(conn, 200, "Mojo Pong")
   end
 
+  # get "site" do
+  #   Site.get_all_sites()
+  #   |> clean_query()
+  #   |> case do
+  #     {:ok, content} ->
+  #       send_resp(conn, 200, "[]")
+  #     _ ->
+  #       send_resp(conn, 500, "Unable to get sites")
+  #   end
+  # end
+
+  # This route can be hit when forward slashes are encoded as %2F
+  post "/site/:name/:endpoint" do
+    case res = Site.create_site(conn.params["name"], conn.params["endpoint"]) do
```

Comment:
All this "route handler" code shouldn't live in the router.  The router file should ideally do minimal processing and simply route requests.  The actual application/business logic that processes the request itself should live in it's own applicable app modules.